### DISCUSSION TITLE
rb2s-install/

### DISCUSSION THREAD

**Discussion by giscus** [Micraow/chat#1] [Announcements]
# rb2s-install/

Rb2S OS看起来很棒，下面这篇文章教你如何将其安装到你的电脑或移动硬盘上 - Micraow - Peng's Blog

https://pengs.top/rb2s-install/

**Comment by Micraow** [comment]
虽然步骤还是有点多，但比从0开始肯定好太多了

**Comment by Micraow** [comment]
Nvidia用户还建议看这个：

https://wiki.archlinuxcn.org/wiki/PRIME

对了，其实我自己的电脑上就是双系统，也没出什么问题，自己觉得水平够的话，你也可以尝试，不复杂，和一般的安装差别可能就在于EFI分区这里留意一下， **还有快速启动一定要关**
